core/src/test/java/com/scalar/db/storage/common/checker/OperationCheckerTest.java

```diff
@@ -537,7 +537,7 @@ public void whenCheckingScanOperationWithInvalidOrderings_shouldThrowIllegalArgu
     Key startClusteringKey = new Key(CKEY1, 2, CKEY2, "val1");
     Key endClusteringKey = new Key(CKEY1, 2, CKEY2, "val9");
     List<String> projections = Arrays.asList(COL1, COL2, COL3);
-    int limit = -10;
```

Comment:
The unit test goal is to validate that a wrong ordering triggers an `IllegalArgumenException`
```
.withOrdering(new Scan.Ordering(CKEY1, Scan.Ordering.Order.DESC))
.withOrdering(new Scan.Ordering(CKEY2, Scan.Ordering.Order.DESC))
 ```
 Though, by mistake, the limit was set to a negative value, throwing an `IllegalArgumenException` before `operationChecker.check(scan)` reaches the step that validates the ordering. This made the test pass for the wrong reason. 
 With a valid positive limit, the test will now validate the wrong ordering.